src/views/Accessibility/AccessibilityRequest/Documents/New/index.tsx

```diff
@@ -215,6 +215,7 @@ const New = () => {
                           onChange(e);
                           setFieldValue('file', e.currentTarget?.files?.[0]);
                         }}
+                        accept=".pdf,.docx,.xlsx"
```

Comment:
What about `.doc` and `.xls`?

Reply:
I tested this and it doesn't allow the user to add `.doc` or `.xls`.